src/components/dropdown-menu/DropdownMenuItemButtonCopy.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { IconCopy } from '..';
+import {
+  DropdownMenuItemButtonBase,
+  DropdownMenuItemButtonBaseProps,
+} from './DropdownMenuItemButtonBase';
+
+interface Props
+  extends Omit<DropdownMenuItemButtonBaseProps, 'isActivable' | 'isActive' | 'prefix'> {}
+
+export function DropdownMenuItemButtonCopy(props: Readonly<Props>) {
```

Comment:
The copy button also has some special behavior, like a copy to clipboard effect some tooltip displayed when copied. On SC I see:

```
interface ItemCopyProps {
  children?: ReactNode;
  className?: string;
  copyValue: string;
}

export function ItemCopy(props: Readonly<ItemCopyProps>) {
  const { children, className, copyValue } = props;
  const [copySuccess, handleCopy] = useCopyClipboardEffect(copyValue);
  return (
    <Tooltip content={translate('copied_action')} isOpen={copySuccess} side={TooltipSide.Left}>
      <li className="dropdown-menu-copy" role="none">
        <ItemButtonStyled
          className={classNames('fs-exclude', className)}
          onClick={handleCopy}
          role="menuitem">
          {children}
        </ItemButtonStyled>
      </li>
    </Tooltip>
  );
}
```

Reply:
cutting this branch for now, the special behavior will be implemented later